src/Logging/PsrLoggerTrait.php

```diff
@@ -34,6 +34,16 @@
     protected abstract function getLogger();
 
     /**
+     * Return common labels for each log entry.
+     *
+     * @return array
+     */
+    protected function getLabels()
+    {
+        return [];
```

Comment:
Should the `PsrLogger` also use the metadata provider? The `PsrBatchLogger` is providing labels, but the non-batch one isn't.